### PR TITLE
Audit search results table

### PR DESCRIPTION
**Pull request by danielnaab**
Implements issue(s) #20, #24, #27, #30

Changes proposed in this pull request:
- Search UI implemented
- Add FAC tables
- Add download management task for all tables

### CODE REVIEW

File: distiller/audit_search/forms.py [hunk 1]
```diff
@@ -98,24 +100,34 @@
...
+        AGENCIES_BY_PREFIX[prefix].append(agency_name)
 
 
 class AgencySelectionForm(forms.Form):
     agency = forms.ChoiceField(choices=AGENCY_CHOICES)
     sub_agency = forms.ChoiceField(required=False)
+    audit_year = forms.ChoiceField(
+        choices=(('', ''),) + tuple(
+            (year, year)
+            for year in range(datetime.now().year, 1996, -1)
```
[the remark this answers is not in the document]
Comment (danielnaab) [reply distiller/audit_search/forms.py]: Ultimately I think we'll just be loading more recent data, but 1996 the first audit year of data available from the FAC.

**Comment by danielnaab** [comment]
> Some paths here seem to be hard-coded to `/Users/dan`; can we change this? Would like to be able to run on my machine locally before approving.

I've been making these more production-ready as part of #30. Would you like me to pull those commits onto this branch?

File: distiller/audit_search/forms.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (bpdesigns) [reply distiller/audit_search/forms.py]: The earliest fiscal year we will want to pull from is 2016

**Comment by bpdesigns** [comment]
@danielnaab  Created  #33 and #34 for more specific info on the findings. I'm on the hook for updating the UI sketches tomorrow morning for displaying the findings text.

**Comment by cantsin** [comment]
> I've been making these more production-ready as part of #30. Would you like me to pull those commits onto this branch?

Yes, please! That helps me with running this code locally. As you'd expect, I get this error when trying to `load_single_audit_db`:

```
FileNotFoundError: [Errno 2] No such file or directory: '/Users/dan/src/10x/fac-distiller/imports/cfda.txt'
```

**Pull request merged**